script/release

```diff
@@ -1,7 +1,28 @@
 #!/bin/sh
-#push a release to branch asf-site
 
+## publish website steps
+CUR_DIR=$PWD
 
-bundle update
+## downlowd code
+git clone https://github.com/apache/servicecomb-website.git servicecomb-website-master
+cp -R servicecomb-website-master servicecomb-website-publish
+git clone https://github.com/huaweicse/servicecomb-java-chassis-doc.git 
 
-git push origin asf-site
+## build source
+cd $CUR_DIR/servicecomb-website-master
+git checkout -f master
+bundle exec jekyll build
+
+## copy complied files
+cd $CUR_DIR/servicecomb-website-publish
+git checkout -B liubao -t origin/asf-site 
```

Comment:
It can be changed to the website repo and the committer has the right to push the change.

Reply:
The push step is commented, and the publisher should use this step to commit changes and create a PR.